platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/CertificateCommand.java

```diff
@@ -163,11 +167,19 @@ public static String configureDemoCertWithDN(String[] dn, @Nullable String[] sub
     if (subjectAltNames != null) {
       csr.addSubjectAlternativeNames(subjectAltNames);
     }
-    RDN[] rdns = csr.getSubjectName().getRDNs(BCStyle.CN);
 
+    // Required for Chrome support (DDF-3104)
+    String commonName =
+        Arrays.stream(dn)
+            .filter(str -> str.startsWith("cn="))
+            .findFirst()
+            .orElseThrow(() -> new IllegalArgumentException("CN is a required attribute on the DN"))
+            .split("=")[1];
+    csr.addSubjectAlternativeNames(String.format("DNS:%s", commonName));
```

Comment:
❓ Same question as above

Reply:
Per @coyotesqrl's comment below, I think the SANs are maintained in a set so no duplication will occur.

Reply:
🤦‍♂️ I added that code